### ISSUE TITLE
Accessibility compliance auto-auditing report

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1071]
These things were suggested by our accessibility office to help communicate the potential difficulty in taking a course. Some of these things can be generated in an automated manner (like seat time).
- perform a bulk audit of a course which runs the functions in elmsln_ecd to batch send things to ECD for analysis
- Seat time (duration of videos, readings, etc. time needed to sit still in a seat to absorb content)
- Visiting external websites (most often an issue for visually impaired students, one person in the meeting said they just worked with a course that pretty much consisted completely of external links, some 300+ external sites)
- Sequential vs. non-sequential course structure: does the information need to be learned sequentially, or is it more topic-based?
- Proctored exams?
- Sensitive topics (potential triggers for students with PTSD, etc.)
- Self-recording (potential issue for students with autism, etc.)
- Synchronous course work
- Shared media (student-generated media that may not be accessible, or may be hosted on an external platform such as google docs or yammer. How is accessibility handled? Will students be required to review other students' work?)
- Group work (potential issue for students with autism, etc)
- Visual content
- Audio content
- Extensive reading and writing (issue for students unable to type or who need to use screen readers)
- Use of eReserves (issue for students with visual impairments)
- Use of PDFs

**Comment by mgifford** [comment]
What happened to these issues?  Have you looked at aXe-core? PDFs should just be replaced with EPUB3.

**Comment by btopro** [comment]
still sitting here; idea was to generate a single page report that had all this data pulled together

**Comment by mgifford** [comment]
That would be pretty cool!  Building on some of the stuff that others is doing is probably a good place to start. Combining it all will be tricky, but doable.

**Comment by btopro** [comment]
indeedie. HAX should hopefully eliminate a lot the above (over time). aXe-core is super interesting I've not seen that before. @heyMP and I really need to build something like that into our element test bed (aka actually have a test bed / CI routine) for our 170+ elements out there now https://github.com/LRNWebComponents
